lib/active_merchant/billing/gateways/braintree/token_nonce.rb

```diff
@@ -34,9 +34,9 @@ def create_token_nonce_for_payment_method(payment_method)
         return token, message
       end
 
-      def client_token
-        base64_token = @braintree_gateway.client_token.generate
-        JSON.parse(Base64.decode64(base64_token))['authorizationFingerprint']
+      def client_token(options = {})
+        base64_token = @braintree_gateway.client_token.generate({ merchant_account_id: options[:merchant_account_id] || @options[:merchant_account_id] }.compact)
```

Comment:
What is this generate method that is being called here?

Reply:
This is a call to the SDK gateway, here the documentation: https://developer.paypal.com/braintree/docs/reference/request/client-token/generate/ruby#client-token:-generate

Reply:
Thank you that is super helpful! I'll finish my review now!